test/acceptance/features/examples.feature

```diff
@@ -1,4 +1,5 @@
 @examples
+@openshift
```

Comment:
why this change?

Reply:
Both scenarios are openshift dependent and last scenario was missing the tag openshift. so instead of adding on two scenarios i added it to the feature file itself

Reply:
see above